www/js/app.js

```diff
@@ -218,19 +220,88 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
+        $('#articleContent').removeClass("fadeIn_R").removeClass("fadeOut_L");
+    }
+    function addAnimation(section){
+            if(section == "home") { 
+                if(!$("#configuration").is(":hidden")){
+                    $('#configuration').addClass("fadeOut_R");
+                    setTimeout(function() {
+                        $('#configuration').hide();
+                    }, 200); 
+                }
+                if(!$("#about").is(":hidden")){
+                    $('#about').addClass("fadeOut_R");
+                    setTimeout(function() {
+                        $('#about').hide();
+                    }, 200); 
+                }
+                $('#articleContent').addClass("fadeIn_R");
+                setTimeout(function() {
+                    $('#articleContent').show();
+                }, 200);
+            }
+            else if(section == "config"){
+                if(!$("#about").is(":hidden")){
+                    $('#about').addClass("fadeOut_R");
+                    $('#configuration').addClass("fadeIn_R");
+                    setTimeout(function() {
+                        $('#about').hide();
+                    }, 200); 
+                }
+                if(!$("#articleContent").is(":hidden")){
+                    $('#articleContent').addClass("fadeOut_L");
+                    $('#configuration').addClass("fadeIn_L");
+                    setTimeout(function() {
+                        $('#articleContent').hide();
+                    }, 200);
+                }
+                setTimeout(function() {
+                    $('#configuration').show();
+                }, 200); 
+            }
+            else if(section == "about"){
+                if(!$("#configuration").is(":hidden")){
+                    $('#configuration').addClass("fadeOut_L");
+                    setTimeout(function() {
+                        $('#configuration').hide();
+                    }, 200); 
+                }
+                if(!$("#articleContent").is(":hidden")){
+                    $('#articleContent').addClass("fadeOut_L");
+                    setTimeout(function() {
+                        $('#articleContent').hide();
+                    }, 200); 
+                }
+                $('#about').addClass("fadeIn_L");
+                setTimeout(function() {
+                    $('#about').show();
+                }, 200);     
+            }
+        
+    }
     $('#btnHome').on('click', function(e) {
         // Highlight the selected section in the navbar
         $('#liHomeNav').attr("class","active");
         $('#liConfigureNav').attr("class","");
         $('#liAboutNav').attr("class","");
         $('.navbar-collapse').collapse('hide');
         // Show the selected content in the page
-        $('#about').hide();
-        $('#configuration').hide();
+            removeClasses();
+        if(params.showUIAnimations){ 
+           addAnimation("home");
+        }
+        else{
```

Comment:
Add spaces between if statements and parentheses, and also (as mentioned earlier) in front of opening braces.

Reply:
Ya ... I will make this sure